lib/deps/request-browser.js

```diff
@@ -214,9 +212,11 @@ function xhRequest(options, callback) {
 }
 
 module.exports = function(options, callback) {
-  if (typeof XMLHttpRequest === 'undefined' && !options.xhr) {
-    return fetchRequest(options, callback);
-  } else {
+  // in Chromium 41, XMLHttpRequest is an object instead of a function, so
```

Comment:
If XMLHttpRequest is an ``'object'``, you can't do ``new XMLHttpRequest()`` (that's a ``TypeError: object is not a function``). That happened in Chromium 41 inside a service worker, so I decided to check for ``'function'`` instead.

The comment was written with the 'old' situation in mind, not the new one. I agree it's confusing. It's probably best to just remove the comment completely. Does that work for you, @KlausTrainer?